### ISSUE TITLE
Video tutorial please

### ISSUE THREAD

**Issue by rahul81281** [carpedm20/LINE#168]
Can you do full video tutorial it will help alot peoples.

**Comment by jayhot1** [comment]
@rahul81281 agreed mate we need to learn @carpedm20  please can you please giv us a tutorial video to how to make a line bot protection bot please?

**Comment by crazy-ace003** [comment]
this is way outdated, carpedm20 stopped updating it because line sent him a cease and desist warning.
